### ISSUE TITLE
Implement debug disabling via features

### ISSUE THREAD

**Issue by manio** [zonyitoo/rust-ini#44]
Hi!
Could you please implement similar functionality as in this commit:
https://github.com/sfackler/rust-postgres/commit/d99843ba917902079952cd8178fa300f54a32436

This way your code will not flood my log with lines like:
```
2018-02-16, 06:44:13.251 [DEBUG] ini::ini: line:0, col:1
2018-02-16, 06:44:13.251 [DEBUG] ini::ini: Got section: postgres
2018-02-16, 06:44:13.251 [DEBUG] ini::ini: line:1, col:1
2018-02-16, 06:44:13.251 [DEBUG] ini::ini: parse comment
2018-02-16, 06:44:13.251 [DEBUG] ini::ini: line:2, col:1
2018-02-16, 06:44:13.251 [DEBUG] ini::ini: Got key: host
2018-02-16, 06:44:13.251 [DEBUG] ini::ini: line:2, col:5

```

**Comment by zonyitoo** [comment]
Maybe I can just log it to TRACE level?

**Comment by zonyitoo** [comment]
Will be released in v0.10.3

**Comment by manio** [comment]
Works great, thank you :)
